Testscripts/Windows/STOR-VHDX-RESIZE-GROWFS.ps1

```diff
@@ -56,10 +86,9 @@ Function Set-HardDiskSize
 	}
 
 	Write-LogInfo "Check if the guest detects the new space"
-	$sd = "sdc"
-	if ( $controllerType -eq "IDE" ) {
-		$sd = "sdb"
-	}
+	$deviceName = Get-DeviceName -ip $ip -port $port
+	Write-LogInfo "The disk device name: $deviceName"
+	$sd = "$deviceName" -replace "/dev/",""
```

Comment:
any reason why you strip down the /dev/ tag here but add it below in the $ret command?

Reply:
The var $sd is used in line 94 and 95.